draft-pda-protocol.md

```diff
@@ -1297,6 +1319,28 @@ participating aggregators stored unencrypted input shares on the same cloud
 object storage service, then that cloud vendor would be able to reassemble all
 the input shares and defeat privacy.
 
+#### Anti-replay {#anti-replay}
+
+Using a report in multiple batches, or multiple times within a single batch, is
+considered a privacy violation, since it may leak more information about that
+report than intended by the PDA protocol. (For example, this may violate
+differential privacy.) To mitigate this issue, the core specification imposes an
```

Comment:
```suggestion
report than intended by the PDA protocol. For example, this may violate
differential privacy. To mitigate this issue, the core specification imposes an
```